proconio/src/source/tokens.rs

```diff
@@ -0,0 +1,63 @@
+use std::{iter::Peekable, ptr::NonNull, str::SplitWhitespace};
+
+pub(super) struct Tokens {
```

Comment:
Because `pub` items are easy to be published like this. 
(This technique is used for so-called ["sealed traits"](https://rust-lang.github.io/api-guidelines/future-proofing.html#sealed-traits-protect-against-downstream-implementations-c-sealed))

```diff
--- a/proconio/src/source/mod.rs
+++ b/proconio/src/source/mod.rs
@@ -57,6 +57,8 @@ use std::fmt::Debug;
 use std::io::BufRead;
 use std::str::FromStr;

+pub use self::tokens::Tokens;
+
 pub mod line;
 pub mod once;
 mod tokens;
```

![image](https://user-images.githubusercontent.com/14125495/230739579-7505f5f8-a922-4302-83f8-4785fe656139.png)

Reply:
Understood. Thanks.